frontend/app/mocks/subscription-api.server.ts

```diff
@@ -229,7 +240,8 @@ export function getSubscriptionApiMockHandlers() {
       return HttpResponse.json({ confirmCodeStatus: 'mismatch' }, { status: 200 });
```

Comment:
The actual API currently returns a 400 if there is no confirmation code match or if the confirmation code expire - no need for `confirmCodeStatus`

Reply:
fixed